packages/blue-sdk/src/market/MarketUtils.ts

```diff
@@ -518,11 +518,7 @@ export namespace MarketUtils {
     },
     marketParams: { lltv: BigIntish },
   ) {
-    borrowShares = BigInt(borrowShares);
-    market.totalBorrowShares = BigInt(market.totalBorrowShares);
-    if (borrowShares === 0n || market.totalBorrowShares === 0n) return null;
-
-    const borrowAssets = toBorrowAssets(borrowShares, market);
+    const borrowAssets = toBorrowAssets(BigInt(borrowShares), market);
```

Comment:
```suggestion
    const borrowAssets = toBorrowAssets(borrowShares, market);
```

not needed either (but not blocking)